scss/civicrm/campaign/_campaign.scss

```diff
@@ -0,0 +1,43 @@
+.page-civicrm-campaign {
+  .crm-content-block {
+    @include box-shadow($box-shadow);
+
+    .dataTables_wrapper,
+    .crm-form-block {
+      box-shadow: none;
+    }
+
+    .dataTables_wrapper {
+      margin-bottom: 0;
+
+      > table {
+        thead th:nth-child(2),
+        tbody td:nth-child(2) {
+          padding-left: 20px;
```

Comment:
why just for this page?

Reply:
First td should have padding-left in normal case but, in this page first td is hidden so, we need to target second one.